vuepress/docs/next/tutorials/consume/high-avail-tutorial.md

```diff
@@ -1,33 +1,17 @@
-# Caching and Clustering
+---
+sidebarDepth: 2
+---
 
-The tutorial below covers the basic steps to setup and validate a clustered instance of the Entando App Engine.
-
-::: tip NOTE
-When building your deployment architecture, it is important to review your goals, hardware, networking and application-specific setup. You must also optimize App Engine deployment for your environment. The configurations and tests below do not address every type of application or deployment, but they can be used as building blocks to create a deployment architecture that works for your application.
-:::
-
-## Storage Requirements for Clustered Entando Apps
-
-In order to scale an Entando Application across multiple nodes, you must provide a storage class that supports
-a `ReadWriteMany` access policy. There are many ways to accomplish this, including using dedicated storage providers like GlusterFS. Cloud Kubernetes providers also offer clustered storage options specific to their implementation, such as Google Cloud File in GKE or Azure Files in AKS.
+# High Availability on Entando
 
-
-You can use two different storage classes for your clustered vs. non-clustered storage if your default class doesn't support `ReadWriteMany`. Add the following properties to your ConfigMap for the operator:
-
-```
-entando.k8s.operator.default.clustered.storage.class: [your clustered RWX storage class]
-entando.k8s.operator.default.non.clustered.storage.class: [your RWO storage class]
-```
-
-Set both values to the appropriate storage class for your configuration.
+To build applications on Entando for high availability (HA), it is important to review your goals, hardware, networking, and application-specific setup as well as optimize App Engine deployment for your environment. To this end, the configurations and tests below can be used as building blocks to create a deployment architecture that promotes HA for your application in most situations. These are the basic steps to setup and validate a clustered instance of Entando App Engine, along with the configuration for Redis to support that instance.
```

Comment:
can be condensed/clarified a little... 

1st sentence: 
"To build applications on Entando for high availability (HA), it is important to review your goals, hardware, networking, and application-specific setup as well as optimize App Engine deployment for your environment."

- you don't need to review/optimize anything to build an HA app - it's good practice, not required
- "for your environment" is implied and can be omitted (lengthens sentence without adding value)
- "as well as" reads as an afterthought when the review and optimization should be on equal footing
- reviewing your goals/hardware/setup/networking is only the first part of the complete action; the second is to use that review to optimize whatever you're doing/building - in this case, your deployment. (reviewing by itself doesn' accomplish anything unless that info is used)

--> "When building high availability (HA) Entando Applications, it is best practice to review your goals, hardware, networking, and application-specific setup to optimize your App Engine deployment."

2nd sentence:
- "To this end" doesn't add value; the first sentence doesn't need a bridge to the second - the relationship is clear
- "in most situations" sits awkwardly as a trailing qualifier and isn't necessary - "can" covers the possibility but doesn't make guarantees
- "...for your application" is also not necessary - that is the only thing HA would apply to (also repetitive)

--> "The configurations and tests below can be used as building blocks to create a deployment architecture that promotes HA."

3rd sentence:
- "These are..." - i'm expecting a list of steps and for this sentence to end with : 
- set up (used as verb, not noun)
- **the** Entando App Engine ("the" always precedes app builder, app engine, etc.)